src/main/java/com/shapesecurity/shift/es2016/parser/ParserWithLocation.java

```diff
@@ -34,6 +59,11 @@ public Module parseModule(@Nonnull String text) throws JsError {
 		return this.locations.get(node);
 	}
 
+	@Nonnull
+	public ImmutableList<Comment> getComments() {
+		return this.comments.reverse();
```

Comment:
Why does the order matter? If they want it reversed, they can reverse it.

Reply:
Consistency with shift-parser-js, mainly.

Reply:
I would say that order just isn't guaranteed. A responsible consumer would be sorting them by offset anyway. Nobody is promising them anything about order.

Reply:
If we expect responsible consumers to be sorting them by offset, we should just sort them by offset ourselves, and guarantee that. Which is the current behavior.

Reply:
I mean a consumer who wants them sorted. There's plenty of consumers who won't care about ordering.